solutions/devsprint-bruno-ramos-5/GitHubApp/Screens/List/ListView.swift

```diff
@@ -67,25 +67,30 @@ private extension ListView {
 }
 
 extension ListView {
-
+    
     func updateView(with repositories: [String]) {
-
+        
         self.listItems = repositories
         self.tableView.reloadData()
     }
 }
 
-extension ListView: UITableViewDataSource {
-
+extension ListView: UITableViewDataSource, UITableViewDelegate {
+    
     public func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-
+        
         return self.listItems.count
     }
-
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        return 62
+    }
+    
     public func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-
-        let cell = tableView.dequeueReusableCell(withIdentifier: self.listViewCellIdentifier)!
-        cell.textLabel?.text = self.listItems[indexPath.row]
+        
+        let cell = tableView.dequeueReusableCell(withIdentifier: listViewCellIdentifier, for: indexPath) as! RepositoryViewCell
+        
+        cell.updateCell(configuration: RepositoryViewCellConfiguration.init(repositoryName: self.listItems[indexPath.row], repsitoryOwnerName: "Owner"))
```

Comment:
Não é necessário você explicitar o método init juntamente com o escopo, nesse caso para dimiuir a linha do código, é possível fazermos de duas formas:
```suggestion
        cell.updateCell(configuration: .init(repositoryName: self.listItems[indexPath.row], repsitoryOwnerName: "Owner"))
```

Reply:
Ou
```suggestion
        cell.updateCell(configuration: RepositoryViewCellConfiguration(repositoryName: self.listItems[indexPath.row], repsitoryOwnerName: "Owner"))
```

Reply:
Alterado